### PR TITLE
Jobseeker without casemanager 315

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#315
Pagination in Agency people homepage : JobSeeker's without casemanager

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
closed

**Comment by vijayakarumudi** [comment]
close the PR

**Comment by vijayakarumudi** [comment]
close the PR

**Comment by vijayakarumudi** [comment]
PR for review.

File: app/views/agency_people/home.html.haml [hunk 1]
```diff
@@ -37,6 +37,7 @@
   - if @js_without_cm.empty?
     %i All job seekers have a Case Manager at the moment.
   -else
-    .col-sm-9
-    = render partial: 'job_seekers' , :locals => {jobseekers: @js_without_cm}
+    .pagination-div{id: "agency_person-#{@people_type_js_without_cm}",
```
Comment (patmbolger) [inline app/views/agency_people/home.html.haml]: The id is incorrect.  This line should be:
```
.pagination-div{id: "agency_person-#{@people_type_without_cm}",
```

The controller sets the instance var `@people_type_without_cm` (there is no instance var `@people_type_js_without_cm`)

**Comment by patmbolger** [comment]
Reviewed PR.  When I looked for the "job seekers without case manager" table on the agency person home page I did not see that (I *did* see "job seekers without job developer").

When I looked at the HTML I saw this:
```
<h3>Your Job Seekers (as job developer)</h3>
<div class='pagination-div' data-url='/agency_people/2/list_js_jd/jobseeker-jd' id='agency_person-jobseeker-jd'></div>
<h3>Your Job Seekers (as case manager)</h3>
<div class='pagination-div' data-url='/agency_people/2/list_js_cm/jobseeker-cm' id='agency_person-jobseeker-cm'></div>
<h3>Job Seekers without a Job Developer</h3>
<div class='pagination-div' data-url='/agency_people/2/list_js_without_jd/jobseeker-without-jd' id='agency_person-'></div>
<h3>Job Seekers without a Case Manager</h3>
<div class='pagination-div' data-url='/agency_people/2/list_js_without_cm/jobseeker-without-cm' id='agency_person-'></div>
```
Note the id's for the last two divs are the same (`id='agency_person-'`).  This explains why we are seeing "js without jd" but not "js without cm", since the former appears first in the page.

So, the root cause is that the id's are not set up correctly in the view.  Please see code comments.

File: app/views/agency_people/home.html.haml (hunk 1 above)
Comment (vijayakarumudi) [inline app/views/agency_people/home.html.haml]: I fixed those lines.

**Comment by vijayakarumudi** [comment]
Updated PR for review.

**Comment by patmbolger** [comment]
Reviewed PR again .... looks good.

**Comment by joaopapereira** [comment]
Looks good, going to merge

**Pull request merged**